packages/transmute-adapter-firestore/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "transmute-adapter-firestore",
+  "version": "0.1.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest"
+  },
+  "author": "",
+  "license": "MIT",
+  "devDependencies": {
+    "firebase-admin": "^6.0.0",
```

Comment:
no because the library uses dependency injection, it takes a db which is initialized from firebase-admin. so a consumer will have firebase-admin as a peer dependency, but the library technically only needs it for the tests to pass.